### PR TITLE
chore(test): use native promises instead of q on onPrepare

### PR DESCRIPTION
**Pull request by cnishina**

### CODE REVIEW

File: spec/onPreparePromiseConf.js [hunk 1]
```diff
@@ -17,9 +18,10 @@ exports.config = {
 
   baseUrl: env.baseUrl + '/ng1/',
 
-  onPrepare: function() {
-    return q.fcall(function() {
-      browser.params.password = '12345';
-    }).delay(1000);
+  onPrepare: async() => {
+    browser.params.password = '12345';
+    return await new Promise((resolve, _) => {
```
Comment (CrispusDH) [inline spec/onPreparePromiseConf.js]: it could be like:
```
return await new Promise(resolve => {
      setTimeout(resolve, 1000);
    });
```

Comment (cnishina) [reply spec/onPreparePromiseConf.js]: Done.

**Comment by cnishina** [comment]
Reference #4995.

File: spec/onPrepare/asyncstartup.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
-var q = require('q');
```
Comment (heathkit) [inline spec/onPrepare/asyncstartup.js]: Yay!

Comment (cnishina) [reply spec/onPrepare/asyncstartup.js]: Woot!

**Pull request merged**